guides/source/autoloading_and_reloading_constants.md

```diff
@@ -162,13 +162,13 @@ The autoload once paths are managed by `Rails.autoloaders.once`.
 $LOAD_PATH{#load_path}
 ----------
 
-Autoload paths are added to `$LOAD_PATH` by default. However, Zeitwerk uses absolute file names internally, and your application should not issue `require` calls for autoloadable files, so those directories are actually not needed there. You can opt out with this flag:
+Autoload paths are added to `$LOAD_PATH` by default. However, Zeitwerk uses absolute file names internally, and the application should not issue `require` calls for autoloadable files, so those directories are actually not needed there. There is an opt-out flag:
 
 ```ruby
 config.add_autoload_paths_to_load_path = false
 ```
 
-That may speed up legitimate `require` calls a bit since there are fewer lookups. Also, if your application uses [Bootsnap](https://github.com/Shopify/bootsnap), that saves the library from building unnecessary indexes, and saves the RAM they would need.
+That may speed up legitimate `require` calls a bit since there are fewer lookups. Also, if the application uses [Bootsnap](https://github.com/Shopify/bootsnap), that saves the library from building unnecessary indexes, and saves the memory space they need.
```

Comment:
```suggestion
That may speed up legitimate `require` calls a bit since there are fewer lookups. Also, if the application uses [Bootsnap](https://github.com/Shopify/bootsnap), that saves the library from building unnecessary indexes, leading to lower memory usage.
```